zuko-api/app/Http/Resources/V1/PostResource.php

```diff
@@ -22,7 +23,17 @@ public function toArray(Request $request): array
             'created' => $this->created_at,
             'description' => $this->description,
             'cover_image' => $this->cover_image_url,
-            'categories' => CategoryResource::collection($this->categories)
+            'categories' => CategoryResource::collection($this->categories),
+            'url' => $this->getUrl(),
         ];
     }
+
+    protected function getUrl()
+    {
+        try {
```

Comment:
Better to check just if there is any elements in attachments no need to waste memory by throwing exceptions and also having the logic to catch them 
Also as a side note don't catch general exceptions always when catching catch specific exceptions or internal app specific exceptions

Reply:
You can just added the ternary operator here and fallback to empty string 
![image](https://github.com/vegaitglobal/informisi-zene-website/assets/147758140/b6a20f15-8927-4943-8544-a6c574d633e4)
 or check for empty array and fallback to empty string

Reply:
We couldn't use this kind of check because attachement as attachement does not exists in PostResource and every time when i try to check like this got error that it is undefined.

Reply:
the this->load loads the attachments just swap the first 2 lines